app/views/layouts/_signup_modal.html.erb

```diff
@@ -5,17 +5,10 @@
     </button>
   </div>
   <div class="global-signup-modal--inner-a">
-    <%= image_tag(cl_image_path(asset_path("/assets/sloan.png"),
-                                type: "fetch",
-                                width: 300,
-                                crop: "imagga_scale",
-                                quality: "auto",
-                                flags: "progressive",
-                                fetch_format: "auto",
-                                sign_url: true), class: "sloan", alt: "Sloan, the sloth mascot", loading: "lazy") %>
+    <%= image_tag(cloudinary(SiteConfig.mascot_image_url, 300), class: "mascot-image", alt: "Sloan, the sloth mascot", loading: "lazy") %>
```

Comment:
There are several other places in the code that calls "/assets/sloan.png" directly. Should I go ahead and change all the other places to use SiteConfig. I am leaning towards keeping this PR small.

Reply:
good call, let's do one step at a time :)

Reply:
Awesome.